src/rss/views.py

```diff
@@ -51,13 +54,17 @@ def item_link(self, item):
 
 
 class LatestArticlesFeed(Feed):
-    title = "Articles"
     link = "/articles/"
-    description = "Updates on changes and additions to police beat central."
 
     def get_object(self, request, *args, **kwargs):
         return request.journal
 
+    def title(self, obj):
+        return "{} Article Feed".format(obj.name)
+
+    def description(self, obj):
+        return "A feed of the 10 latest articles from {}".format(obj.name)
+
```

Comment:
can `obj` be `None` ? It seems that way from `self.get_object`, in which case it would raise an `AttributeError` here